modules/renter/downloadqueue.go

```diff
@@ -60,6 +60,9 @@ func (r *Renter) DownloadQueue() []modules.DownloadInfo {
 			StartTime:   d.startTime,
 		}
 		downloads[i].Received = atomic.LoadUint64(&d.atomicDataReceived)
+		if d.downloadErr != nil {
+			downloads[i].Error = d.downloadErr.Error()
+		}
```

Comment:
It looks to me like d.downloadErr will always be set by the worker thread before returning from the `Download` call, though this isn't immediately obvious